### PR TITLE
rename rules

### PR DESCRIPTION
**Pull request by FalkF**
closes: #283 
deprecated rule names are corrected at runtime and a deprecation message will be printed.

template-lint-deprecations

### CODE REVIEW

File: lib/get-config.js [hunk 1]
```diff
@@ -276,16 +306,27 @@ function processIgnores(config) {
 function validateRules(config, options) {
   let logger = options.console || console;
   let invalidKeysFound = [];
+  let deprecatedNamesFound = [];
 
   for (let key in config.rules) {
     if (!config.loadedRules[key]) {
-      invalidKeysFound.push(key);
+      if (RULE_NAME_DEPRECATIONS.map(a => a.oldName).indexOf(key) > -1) {
+        const deprecation = RULE_NAME_DEPRECATIONS.find(a => a.oldName === key);
```
Comment (FalkF) [inline lib/get-config.js]: I should move this up and replace the mapping in the if.

**Comment by rwjblue** [comment]
Mind doing a rebase to kick off a new CI job?

**Comment by FalkF** [comment]
@rwjblue done :+1:

**Comment by rwjblue** [comment]
@FalkF - This looks good to me, I seem to have made a conflict though in a recent PR merge. Can you rebase?

Also, to confirm, this is specifically _not_ a breaking change (because we deprecate but still allow the "old" names).

**Comment by rwjblue** [comment]
OK, I rebased this and fixed up an issue with the inline configuration comments not working (when using the now deprecated names).

**Pull request merged**